src/mudlet-lua/lua/geyser/GeyserMiniConsole.lua

```diff
@@ -147,6 +155,21 @@ function Geyser.MiniConsole:getColumnCount()
     return getColumnCount(self.name)
 end
 
+--- Set auto wrap on for the miniconsole
+function Geyser.MiniConsole:setAutoWrap()
+  self.autoWrap = true
+  self:resetAutoWrap()
+end
+
+--- Set the wrap based on how wide the console is
+function Geyser.MiniConsole:resetAutoWrap()
+  local fontWidth, fontHeight = calcFontSize(self.fontSize)
```

Comment:
If you could change this to `self.name` so it'll take the custom font into account (available with https://github.com/Mudlet/Mudlet/commit/1dd7427d6fcf2338e65ccdcd8962da7efab095b4) and apply https://github.com/Mudlet/Mudlet/pull/1708/files#r191332826, it'll be good to go :100: